VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+# STARTTLS Policy List Validation
+
+## MTA-STS
+
+In order to be preloaded onto the STARTTLS policy list, the easiest way to do and maintain this is to have a valid MTA-STS record up for a max-age of at least half a year (10886400) is sufficient to be preloaded on the STARTTLS policy list.
+
+To change your entry on the list, you can serve a valid MTA-STS record with your new expected policy.
+
+To be removed from the list, you can serve a valid MTA-STS record with `mode: none`.
+
+## Manual addition
+
+If you don't have an MTA-STS file up, For your email domain to be eligible for addition to the STARTTLS policy list, the requirements are that your domain:
+
+ * Supports STARTTLS (as a receiving mailserver)
+ * Supplied MX hostnames are valid (DNS lookup from this perspective resolves to hostnames that match the given patterns).
+ * Provides a valid PKI certificate. Validity is defined as in [RFC 3280](https://tools.ietf.org/html/rfc3280#section-6). To clarify:
+    * The certificate's Common Name or a subjectAltName should match the server's MX hostname.
+    * There is a valid chain from the certificate to a root included in [Mozilla's trust store](https://wiki.mozilla.org/CA/Included_Certificates) (available as Debian package [ca-certificates](https://packages.debian.org/sid/ca-certificates)).
```

Comment:
Maybe "There is a valid chain, served by the server, from the certificate to a root..."?